lib/puppet/provider/chocolateyfeature/windows.rb

```diff
@@ -0,0 +1,138 @@
+require 'puppet/type'
+require 'pathname'
+require 'rexml/document'
+
+Puppet::Type.type(:chocolateyfeature).provide(:windows) do
+  confine :operatingsystem => :windows
+  defaultfor :operatingsystem => :windows
+
+  require Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/chocolatey/chocolatey_common'
+  include PuppetX::Chocolatey::ChocolateyCommon
+
+  MINIMUM_SUPPORTED_CHOCO_VERSION = '0.9.9.0'
```

Comment:
The test output goes by so fast, you don't see that we do get a warning for duplicate const.

The problem here we are trying to solve is that different capabilities were introduced in different minor versions and we need to have a bare minimum before we can even run. `source` happens to have the same version as `feature` coincidently, but `source` also tracks more versions. There are many point versions where features (ever use a word too many times and now it loses meaning?) were introduced, so I can easily see us adding more as we add other types. 

I would suggest making a single constant, but it's just coincidence that these two share the same version and we would be back in the same boat. I agree this should be named, but should we also change the other test to align with this?